pkg/client/consumerlister.go

```diff
@@ -0,0 +1,6 @@
+package client
+
+type ConsumerLister interface {
```

Comment:
Underscore `_` separate file names

Reply:
Right now, all files in the repository are not underscore separated. Will involve lot of changes to do across all files.